server.js

```diff
@@ -2,26 +2,41 @@ const express = require("express");
 const path = require("path");
 require("dotenv").config();
```

Comment:
I think best practice is to put dotenv it in Line 1

Ref: https://github.com/motdotla/dotenv#usage

Reply:
perfect, thanks, didin't know that!👍